conbench/app/_plots.py

```diff
@@ -465,20 +486,30 @@ def gen_js_callback_click_on_glyph_show_run_details(repo_string):
 
 
 def time_series_plot(
-    samples: List[HistorySample], benchmark, run, height=380, width=1100
+    samples: List[HistorySample],
+    current_benchmark_result: BenchmarkResult,
+    run,
+    height=380,
+    width=1100,
 ):
-    # log.info("Time series plot for:\n%s", json.dumps(samples, indent=2))
+    # log.info(
+    #     "Time series plot for:\n%s",
+    #     json.dumps([s._dict_for_api_json() for s in samples], indent=2, default=str),
+    # )
```

Comment:
Could we remove this commented code? Or uncomment it?

Reply:
Super happy to remove outcommented code in general because often it's really just an accidental leftover. This one here, though, is a carefully crafted log statement that helps debugging _plots.py whenever I touch it because the input to this function is / was a little unpredictable. The `default=str` guards against situations were objects are in the input are by default not JSON-serializable. I like to have this one here readily available for debugging. It's too verbose though even for DEBUG mode, hence outcommented.